pom.xml

```diff
@@ -63,6 +63,32 @@
                     </execution>
                 </executions>
             </plugin>
+<!--
```

Comment:
Changes in the build should not be part of this PR - unless they are strictly required

Reply:
Done